### PR TITLE
Update the export artifacts page

### PR DESCRIPTION
**Pull request by gabilang**

### CODE REVIEW

File: en/docs/develop/exporting-artifacts.md [hunk 1]
```diff
@@ -1,11 +1,8 @@
 # Exporting packaged Synapse artifacts
```
Comment (chanikag) [inline en/docs/develop/exporting-artifacts.md:1]:
```suggestion
# Export packaged Synapse artifacts
```

**Comment by chanikag** [comment]
Let's rename this to build and export artifacts

and add the option to build and export

**Comment by chanikag** [comment]
Yes let's remove this page. Add proper redirections too, to the [deploy-artifacts/](https://mi.docs.wso2.com/en/latest/develop/deploy-artifacts/) page

**Comment by gabilang** [comment]
Closing this since the content of `Export Artifacts` page has been moved to https://mi.docs.wso2.com/en/latest/develop/deploy-artifacts/#build-and-export-the-carbon-application

**Pull request closed without merging**